llama_stack/cli/stack/run.py

```diff
@@ -15,7 +15,7 @@
 
 REPO_ROOT = Path(__file__).parent.parent.parent.parent
 
-logger = get_logger(name=__name__, category="server")
+log = get_logger(name=__name__, category="server")
```

Comment:
yeah why this spurious change? please move it back to `logger` as the variable name please. 

It is extremely important that you don't add gratuitous changes when making PRs. Stick to the core thesis of the PR.

Reply:
thanks for your review 

i just changed this due to this review https://github.com/meta-llama/llama-stack/pull/2868#discussion_r2254484687

How to proceed ?

Reply:
i am preserving this [commit](https://github.com/meta-llama/llama-stack/commit/f24d136f615b7de3dd1dc2744bf46e4a5d0644c0) here for your ref. 

This commit was using `logger`

Reply:
raised https://github.com/meta-llama/llama-stack/pull/3060 👍🏽